include/clover_field.h

```diff
@@ -258,6 +304,15 @@ namespace quda {
     */
     void prefetch(QudaFieldLocation mem_space, qudaStream_t stream, CloverPrefetchType type,
                   QudaParity parity = QUDA_INVALID_PARITY) const;
+    /**
+      TODO: Doxygen
+    */
+    virtual void copy_to_buffer(void *buffer) const;
+
+    /**
+      TODO: Doxygen
```

Comment:
Already marked as TODO, but just making it here with a github comment to not forget.